openapi.yaml

```diff
@@ -1,190 +1,432 @@
+      summary: List all parks
       tags:
-        - pets
-      description: Get all pets
-      operationId: getPets
+        - parks
+      description: Get all parks
+      operationId: getParks
       parameters:
-        - $ref: '#/parameters/authorization'
-        - $ref: '#/parameters/pageNumber'
-        - $ref: '#/parameters/pageSize'
         - in: query
-          name: species
-          description: Search by pet species
+          name: amenities
+          description: Filter by park amenities (play area, hiking trails, etc.)
+          required: false
+          type: array
+          collectionFormat: csv
+          items:
+            type: string
+            enum:
+              - ballfield
+              - barbeque grills
+              - basketball courts
+              - bike paths
+              - boat ramps
+              - dogs allowed
+              - drinking water
+              - fishing
+              - hiking trails
+              - horseshoes
+              - natural area
+              - offleash dog park
+              - open fields
+              - picnic shelters
+              - picnic tables
+              - play area
+              - restrooms
+              - scenic view point
+              - soccer fields
+              - tennis courts
+              - volleyball
+        - in: query
+          name: parkName
+          description: Filter parks by name
+          required: false
+          type: string
+        - in: query
+          name: ownerId
+          description: Filter parks by owner ID
+          required: false
+          type: string
+        - in: query
+          name: city
+          description: Filter parks by city
+          required: false
+          type: string
+        - in: query
+          name: state
+          description: Filter parks by state
+          required: false
+          type: string
+        - in: query
+          name: zipCode
+          description: Filter parks by zip code
           required: false
           type: string
       responses:
         '200':
           description: Successful response
           schema:
-            $ref: '#/definitions/PetsResult'
+            $ref: '#/definitions/ParksResult'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/ErrorResult'
+    post:
+      summary: Creates a new park
+      tags:
+        - parks
+      operationId: createPark
+      parameters:
+      - $ref: '#/parameters/parkBody'
+      responses:
+        '201':
+          description: Successfully created new park
+          schema:
+            $ref: '#/definitions/ParkResult'
         '400':
-          description: Bad request
+          description: Invalid park object
           schema:
             $ref: '#/definitions/ErrorResult'
         '500':
           description: Internal server error
           schema:
             $ref: '#/definitions/ErrorResult'
```

Comment:
I think this endpoint may also have to respond with a `409`: https://jsonapi.org/format/#crud-creating-responses-409